sdk/python/approzium/__init__.py

```diff
@@ -1,7 +1,21 @@
 import logging
 
+authenticator_addr = "authenticator:1234"
+iam_role = None
 
 logging.basicConfig(
     level=logging.DEBUG,
     format="[%(filename)s:%(lineno)s - %(funcName)10s() ] %(message)s",
 )
+
+
+def set_authenticator(new_authenticator):
+    global authenticator_addr
+    authenticator_addr = new_authenticator
+
+
+def set_iam_role(new_iam_role):
+    # this function tells Approzium to assume an IAM role and use that as its identity
```

Comment:
this is just a temporary comment. since this method is user-facing, adding proper doc-string should be done. this will be part of the documentation issue.